src/main/java/org/fcrepo/spec/testsuite/TestSuiteGlobals.java

```diff
@@ -68,12 +72,33 @@ public static String containerTestSuite(final String baseurl, final String user,
                                         .when()
                                         .post(base);
         if (res.getStatusCode() == 201) {
+            cleanupManager.setTestContainerUrl(containerUrl);
             return containerUrl;
         } else {
             return base;
```

Comment:
What do you think about throwing an exception if `res.getStatusCode != 201`?

Reply:
That seems preferable to me, but wasn't sure if there was some expectation that this might fail for other implementations

Reply:
Creating a top-level container seems like a reasonable assumption for any implementation.